modules/ocf/manifests/packages/chrome.pp

```diff
@@ -1,4 +1,8 @@
 class ocf::packages::chrome {
+  class { 'ocf::packages::chrome::apt':
```

Comment:
Yeah, I understand now. Thanks for clarification. I set up that `stage => first` hack 4 years ago and it's ugly.